cmd/secrets_delete_test.go

```diff
@@ -54,8 +54,8 @@ func TestSecretsDelete(t *testing.T) {
 		"valid and invalid secrets": {
 			serverResponses: map[string]ServerResponse{
 				secretsDeleteServerPath: {
-					statusCode: http.StatusNotFound,
-					body:       `["secret1"]`,
+					statusCode:   http.StatusNotFound,
+					responseFile: "secrets_delete_not_found.json",
```

Comment:
What do you think about renaming the file as `secrets_delete_single.json`?

Reply:
Could be, that is being used for a not found secret but the response has the exact same structure